service/handlers.go

```
@@ -3,40 +3,60 @@ package service
 import (
 	"encoding/json"
 	"fmt"
-	log "github.com/Sirupsen/logrus"
-	"github.com/rancher/webhook-service/drivers"
+	"github.com/Sirupsen/logrus"
+	// "github.com/rancher/go-rancher/api"
 	"github.com/rancher/webhook-service/webhooks"
 	"io/ioutil"
 	"net/http"
 )
 
-type WebhookRequest map[string]interface{}
-
 func ConstructPayload(w http.ResponseWriter, r *http.Request) {
```

Should return an error. See my comment in routes.go for more details.